mime-types-redirector/lib/mime-types.rb

```diff
@@ -0,0 +1,13 @@
+require 'mini_mime'
+
+module MIME
+  class Types
+    def self.[] type
+      Array.wrap MiniMime.lookup_by_content_type(type)
+    end
+
+    def self.type_for filename
+      Array.wrap MiniMime.lookup_by_filename(filename)
```

Comment:
I think I was right the first time... see https://github.com/ManageIQ/manageiq/pull/14525#issuecomment-289531404